SWLOR.Game.Server/Conversation/HoloCom.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using NWN;
+using static NWN._;
+using SWLOR.Game.Server.Data.Entity;
+using SWLOR.Game.Server.GameObject;
+using SWLOR.Game.Server.Service;
+
+using SWLOR.Game.Server.ValueObject.Dialog;
+
+namespace SWLOR.Game.Server.Conversation
+{
+    public class HoloCom: ConversationBase
+    {
+        public override PlayerDialog SetUp(NWPlayer player)
+        {
+            PlayerDialog dialog = new PlayerDialog("MainPage");
+
+            DialogPage mainPage = new DialogPage("Holocom Main Menu:\n\n" + 
+                                                 "How may we help you?"
+                                                ); // Responses dynamically generated            
+
+            dialog.AddPage("MainPage", mainPage);
+            return dialog;
+        }
+
+        public override void Initialize()
+        {
+            LoadMainMenu();
+        }
+
+        public void LoadMainMenu()
+        {            
+            ClearPageResponses("MainPage");
+
+            string header = ColorTokenService.Green("HoloCom Menu\n\n");
+            SetPageHeader("MainPage", header);
+
+            NWPlayer user = GetPC();
+            Console.WriteLine("LoadMainMenu: user = " + user.Name + ", receiver = " + HoloComService.GetCallReceiver(user).Name);
```

Comment:
Can you swap these to LogginService calls or remove them? Server's gonna get spammed. If you need them for testing that's ok too. Just need to do something about it before we go live with it.

Reply:
Yeah, i thought i removed that already. Woops